platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/Network.java

```diff
@@ -65,4 +66,17 @@ public interface Network {
      * @throws InterruptedException if the thread is interrupted while waiting
      */
     void resume(@NonNull Duration duration) throws InterruptedException;
+
+    /**
+     * Gets the consensus rounds of multiple nodes.
+     *
+     * <p>It is possible to request only the results of a subset of nodes by providing filters.
+     * The filters are applied to the nodes and only the nodes that match the filters are included
+     * in the result. If no filters are provided, all nodes are included in the result.
+     *
+     * @param filters the filters to apply to the nodes
+     * @return the consensus rounds of the node
```

Comment:
Probably a more clear description will be:

`the consensus rounds of the filtered nodes`

Reply:
Good point. Will fix it in an upcoming PR.